app/controllers/v0/post911_gi_bill_statuses_controller.rb

```diff
@@ -78,14 +78,14 @@ def user_json(user)
         participant_id: user.participant_id,
         vet360_id: user.vet360_id,
         ssn: user.ssn,
-        birth_date: iso8601_birth_date(user)
+        birth_date: birth_date_formatted(user.birth_date)
       }.to_json
     end
 
-    def iso8601_birth_date(user)
-      return nil unless user&.va_profile&.birth_date
+    def birth_date_formatted(birth_date)
```

Comment:
Does this method need to live in the controller?

Reply:
It also looks like the same method is repeated in lib/evss/auth_headers.rb and . lib/evss/disability_compensation_auth_headers.rb These repeat methods could probably be pulled out to a shared helper

Reply:
Oh yeah, hmmm, that might not be a bad idea. Basically you can call `user.birth_date` directly if you want a birth_date but you don't care much about the format (where the format is some default sensible option). But if you want a specific format, you use the `DateFormatterHelper` or whatever we'd call it